packages/web-runtime/src/store/spaces.ts

```diff
@@ -69,6 +94,58 @@ const actions = {
 
     const spaces = graphResponse.data.value.map((space) => buildSpace(space))
     context.commit('LOAD_SPACES', spaces)
+  },
+  loadSpaceMembers(context, { graphClient, space }) {
+    context.commit('LOAD_SPACE_MEMBERS', [])
```

Comment:
For spaces we have a `CLEAR_SPACES` mutation. Don't know if it's really worth it, but you could introduce the same for space members.